pkg/controller/cassandradatacenter/statefulset.go

```diff
@@ -127,12 +127,12 @@ func newCassandraContainer(cdc *cassandraoperatorv1alpha1.CassandraDataCenter, d
 		Image:           cdc.Spec.CassandraImage,
 		ImagePullPolicy: cdc.Spec.ImagePullPolicy,
 		Args:            []string{OperatorConfigVolumeMountPath},
-		Ports: []corev1.ContainerPort{
-			{Name: "internode", ContainerPort: 7000},
-			{Name: "internode-tls", ContainerPort: 7001},
-			{Name: "cql", ContainerPort: 9042},
-			{Name: "jmx", ContainerPort: 7199},
-		},
+		Ports: ports{
+			{"internode", 7000},
```

Comment:
Since these are used in multiple places, it might make sense to make them constants. Especially since now that we have a `port` type that can be converted to either a `Service...` or `ContainerPort` type, they can be kept in one spot.